examples/testprograms/http/http.star

```diff
@@ -7,7 +7,7 @@ def foo(x):
     return "meow, " + x
 
 def on_http_get():
-    resp, err = http1.get("https://httpbin3212.org/status/404", ak.callopts(catch=True))
+    resp, err = http1.get("https://httpbin.org/status/404", ak.callopts(catch=True))
```

Comment:
Is there any explanation to the user about all this catch/callopts things?

Reply:
@haimzlato will work on it.